environment.yml

```diff
@@ -5,10 +5,10 @@ channels:
   - defaults
 dependencies:
   - graphviz=2.40.1
-  - jinja2=2.11.2
+  - jinja2=3.0
   - networkx=2.5
   - pygments=2.6.1
   - python=3.7.4
-  - snakemake=6.10.0
-  - unzip=6.0
-  - wget==1.20.1
+  - snakemake=7.24.0
```

Comment:
Are _any_ of the dependencies, other than Snakemake and Python, actually _needed_? Did you try that out? I think that most of them would either be installed as part of Snakemake anyway, or they aren't actually needed (at least I wouldn't know where). Adding dependencies that are not needed (for either of these two reasons) just complicate our lives, because we will need to manage upgrading them much more often, and upgrades will also be more complicated.

Please create an environment with a recent version of Python, say `python=3.10`, from channel `conda-forge`. Then install Snakemake (from channel `bioconda`), without any version hint. Then see if your tests run. If yes, then add the Snakemake version that Conda actually installed to the environment file. If no, then try with a `snakemake=7.24.0` explicitly. If it still does not work, then you should hopefully get some error message that gives you a clue which additional dependency is really required. Add it, try again and so on. Only add what's needed to avoid errors.

Remove channel `defaults`.